### PR TITLE
Fix more styling issues in the nav column 😩

### PR DESCRIPTION
**Pull request by nicoback2**
### Description
-Fix regression I caused where long playlist names were clipped instead of ellipsis-ed (not in prod yet)
-Fix spacing in between playlist links (too big before) to match [these designs](https://www.figma.com/file/u8UHdkD60030aqsRkItWUV/Playlist-Folders?node-id=101%3A141) 
-Consolidate some styles

Screen Shot 2022-03-07 at 7 21 28 PM

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

Tested in browser with long and short playlist/folder names, playlist folder flag on/off, playlist updates/not

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### CODE REVIEW

File: packages/web/src/components/nav/desktop/NavColumn.module.css [hunk 1]
```diff
@@ -280,7 +280,7 @@ svg.logo path {
   /* negative margin so the link hover chip shows up. */
   margin-left: -16px;
   /* left padding is visibly (32px + -16px margin =) 16px to match the left padding of .groupHeader */
-  padding: 6px 0 6px 32px;
+  padding: 5px 0 5px 32px;
```
Comment (raymondjacobson) [inline packages/web/src/components/nav/desktop/NavColumn.module.css:283]: how do we design system this 5px thing!!!!! 🤮

**Review by raymondjacobson — APPROVED** [review]
lgtm!!!

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--fix-more-styles

File: packages/web/src/components/nav/desktop/NavColumn.module.css (hunk 1 above)
Comment (nicoback2) [reply packages/web/src/components/nav/desktop/NavColumn.module.css:283]: `spacing(5/8)` 💩

**Pull request merged**